spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -310,6 +320,52 @@ def _make_figure(metric, fit_results):
     return fname_img
 
 
+def get_data_for_normalization(norm_args):
+    """
+    Get coefficients of multilinear regression for CSA normalization and subject data.
+    Normalization models are under data/csa_normalization_models/
+    Models are generated with https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/blob/master/pipeline_ukbiobank/cli/compute_stats.py
+    # TODO update link with release tag.
+
+    :param norm_args: arguments from -normalize.
+
+    :return data_predictors and data_subject:
+
+    """
+    PREDICTORS_DICT = {'brain-volume': 'brain volume', 'thalamus-volume': 'thalamus volume'}
+    # Select model
+    model_1 = ['sex', 'brain-volume']
+    model_2 = ['thalamus-volume', 'sex', 'brain-volume']
+    if all(item in norm_args for item in model_1):
+        model = 'coeff_brain_sex'
+    elif all(item in norm_args for item in model_2):
+        model = 'coeff_brain_thalamus_sex'
+    else:
+        raise ValueError('Invalid choice of predictors in -normalize. Please specify sex and brain-volume or sex, brain-volume and thalamus-volume')
```

Comment:
We could also name the `.csv` files based on their predictors (sorted alphabetically), for example:

`brain-volume_inter-bv-sex_sex_thalamus-volume.csv`. 

That way, instead of hardcoding the CSV filenames, we can programmatically load the CSV file based on something like: 

```python
path_model = os.path.join(__sct_dir__, 'spinalcordtoolbox', 'data', 'csa_normalization_models', 
                          '_'.join(sorted(data_subject.columns)) + '.csv')
```

I think that would simplify this logic a little further.

Reply:
great suggestion!! Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3515/commits/725758526ad874664a2b83612f0a4d24a569bfce and https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3515/commits/145d7736a356a44a04b0011b10b47ff32cc6f894